pages/common/git-merge-into.md

```diff
@@ -0,0 +1,13 @@
+# git merge-into
+
+> Merge one branch into another branch.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-merge-into>.
+
+- Merge source branch into destination branch:
+
+`git merge-into {{source-branch}} {{destination-branch}}`
+
+- Merge current branch into destination branch:
+
+`git merge-into {{destination-branch}}`
```

Comment:
```suggestion
`git merge-into {{destination_branch}}`
```

Mandatory change: snake case is [required](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming).